nixos/modules/services/backup/borgbackup.xml

```diff
@@ -0,0 +1,227 @@
+<chapter xmlns="http://docbook.org/ns/docbook"
+         xmlns:xlink="http://www.w3.org/1999/xlink"
+         xmlns:xi="http://www.w3.org/2001/XInclude"
+         version="5.0"
+         xml:id="module-borgbase">
+ <title>BorgBackup</title>
+  <para>
+  <emphasis>Source:</emphasis>
+  <filename>modules/services/backup/rsnapshot.nix</filename>
```

Comment:
```suggestion
  <filename>modules/services/backup/borgbackup.nix</filename>
```

Found a mistake here.

Reply:
I don't see any other problem. Once you are done, please squash the commits  (I use `git rebase -i` for this) and I'll merge the PR.

Reply:
Okay. Rebase done. The `git rebase -i `is now much more user friendly. I stemmed away of using it a few years ago, but this time it worked like a charm. Thanks for the tip.